code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -63,7 +63,7 @@
 		to_chat(user, "<span class='warning'>This syringe is broken!</span>")
 		return
 
-	if(user.a_intent == I_HURT && ismob(target))
+	if(user.a_intent == I_HURT && ismob(target) && user.type!=/mob/living/silicon/robot)

```

Comment:
Please use `isrobot(user)`

Reply:
And in the future, should you need to check a type (and a helper doesn't exist for it), use `istype(thing, /path/to/thing/to/check)

Reply:
Done